src/styles.jl

```diff
@@ -194,7 +207,15 @@ function styles_is_float(wb::Workbook, index::Int) :: Bool
             isfloat = true
         elseif numFmtId > 81
             code = styles_numFmt_formatCode(wb, numFmtId)
-            if contains(code, ".0")
+            code = remove_formatting(code)
+
+            floatformats = r"""
+                \.[0#?]|
+                [0#?]e[+-]?[0#?]|
+                [0#?]/[0#?]|
+                %
+                """ix
+            if ismatch(floatformats, code)
```

Comment:
Use `occursin(floatformats, code)` for Julia 1.0 compatibility

Reply:
No need to, since this is not defined in Julia v0.6 .
Don't worry about this for now.